ui/src/api/access/serializer.js

```diff
@@ -0,0 +1,19 @@
+import { isEmpty } from 'lodash';
+
+export function serializeResponse(hit) {
+  const result = {};
+  if (!isEmpty(hit)) {
+    result['user_pid'] = hit.id.toString();
+    result['links'] = hit.links;
+    result['email'] = hit.email;
```

Comment:
Do we always have `email` and `active`? What if we provide another implementation of the factory?